components/Downloads/Release/BitnessDropdown.tsx

```diff
@@ -14,13 +14,21 @@ const parseNumericBitness = (bitness: string) =>
   /^\d+$/.test(bitness) ? Number(bitness) : bitness;
 
 const BitnessDropdown: FC = () => {
-  const { bitness: userBitness } = useDetectOS();
+  const { bitness: userBitness, architecture: userArchitecture } =
+    useDetectOS();
   const { bitness, os, release, setBitness } = useContext(ReleaseContext);
   const t = useTranslations();
 
   // we also reset the bitness when the OS changes, because different OSs have
   // different bitnesses available
-  useEffect(() => setBitness(userBitness), [setBitness, userBitness]);
+
+  useEffect(() => {
+    if (userArchitecture == 'arm' && userBitness == 64) {
```

Comment:
Instead of this, could you create an util function inside the util folder like `getUserBitnessByArchitecture` (similar as the `getNodeDownloadSnippet`

where you have two arguments (architecture and bitness) and then you have if statements for these special cases.

For example, there you could also include the fix for macOS (when on M1 on macOS, it selects x64 instead of ARM64)

So that allows us to easily create a comprehensive list of edge cases

Then on this effect you just do `useEffect(() => setBitness(getUserBitnessByArchitecture(userArchitecture, userBitness), [userArchitecture, userBitness]);`